commit 081a45e53f5151101dc17f92c9b183701dedb8aa (refs/changes/89/5289/4)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-01-02T11:49:41-08:00 (9 months ago)
    
    TOOLS-1619 sdc-image-sync should skip nat instances
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/tools/bin/sdc-image-sync b/tools/bin/sdc-image-sync
index c558b8e4..c55b3a62 100755
--- a/tools/bin/sdc-image-sync
+++ b/tools/bin/sdc-image-sync
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 # This script is intended to copy any changes that a sysadmin might legitimately
@@ -77,7 +77,7 @@ update_sdc_zone_pkgs()
 	    | json -H  \
 	    -e 'this.smartdc_role=(this.tags.smartdc_role || "-")' \
 	    -e 'this.package_name=(this.package_name || "-")' \
-	    -c 'this.tags.smartdc_role' \
+	    -c 'this.tags.smartdc_role && this.tags.smartdc_role!=="nat"' \
 	    -a smartdc_role package_name | \
 	    while read role pkg
 	    do
